jupyter-notebooks/api_guides/features_api/planet_sdk_features_demo.ipynb

```diff
@@ -0,0 +1,742 @@
+   "source": [
+    "`add_items` returned a list of strings, which are the `pl reference ids` of the items we just added to the colleciton. These string are what we can use to as the `geometry` input for Orders and Subscriptions later. Before we order imagery with these, we will first explore the other functionalities of the Features python client."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eaad89f4",
+   "metadata": {},
+   "source": [
+    "____"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3bcf7ccd",
+   "metadata": {},
+   "source": [
+    "### Access your feature Collecitons from the Features API"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e94f5cb5",
+   "metadata": {},
+   "source": [
+    "Basic information (title, description, extent, etc.) about all the feature collections you have access to is available with `list_collections()`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6fff398e",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[[{'id': 'cairo_areas-orELyjq',\n",
+       "   'title': 'cairo_areas',\n",
+       "   'description': 'FeatureCollection of two areas in Cario, Egypt',\n",
+       "   'item_type': 'feature',\n",
+       "   'created': '2025-12-12T18:27:21Z',\n",
+       "   'updated': '2025-12-12T18:34:58Z',\n",
+       "   'extent': {'spatial': {'bbox': [[31.23344423,\n",
+       "       29.99579775,\n",
+       "       31.26630889,\n",
+       "       30.06461941]]}},\n",
+       "   'links': [{'href': 'https://api.planet.com/features/v1/ogc/my/collections/cairo_areas-orELyjq',\n",
+       "     'rel': 'self',\n",
+       "     'title': 'This collection',\n",
+       "     'type': 'application/json'},\n",
+       "    {'href': 'https://api.planet.com/features/v1/ogc/my/collections/cairo_areas-orELyjq/items',\n",
+       "     'rel': 'features',\n",
+       "     'title': 'Features',\n",
+       "     'type': 'application/json'}],\n",
+       "   'feature_count': 2,\n",
+       "   'area': 9890943.929543883,\n",
+       "   'title_property': None,\n",
+       "   'description_property': None,\n",
+       "   'properties': {},\n",
+       "   'permissions': {'can_write': True, 'shared': False, 'is_owner': True},\n",
+       "   'ownership': {'owner_id': '<redacted>', 'org_id': '<redacted>'},\n",
+       "   'pl:ref': 'pl:features/my/cairo_areas-orELyjq'}]]"
+      ]
+     },
+     "execution_count": 71,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "collections = pl.features.list_collections()\n",
+    "collections_list = [collection for collection in collections]\n",
+    "collections_list"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d6864356",
+   "metadata": {},
+   "source": [
+    "Individual Collection items can be listed with `list_items()`. You can then use the item ids' returned from the items to access the individual items.\n",
```

Comment:
Spelling/grammer: `ids'` -> `ids`

Also, you can probably remove "from the items" from this sentence, as it's already implied.